Makefile

```diff
@@ -71,6 +71,7 @@ validate: bin/golangci-lint bin/wwhrd
 generate: bin/operator-sdk bin/openapi-gen bin/client-gen bin/informer-gen bin/lister-gen
 	./bin/operator-sdk generate k8s
 	./bin/operator-sdk generate crds
+	cp -f deploy/crds/*_crd.yaml chart/datadog-operator/templates/
```

Comment:
why this? we use a sym link https://github.com/DataDog/datadog-operator/blob/master/chart/datadog-operator/templates/datadoghq.com_datadogagents_crd.yaml

Reply:
Sorry.
It’s because in the `extendeddaemonset` repo, the CRD files were copied from `deployed` to `chart` and they were not synchronized.
That’s why I thought that helm was not honoring symlinks.
So, I changed to also use symlinks in `extendeddaemonset`: DataDog/extendeddaemonset#11.

Reply:
thanks for fixing the `extendeddaemonset` 🙇